test/Microsoft.Data.Analysis.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Add a comment here about why we are doing this.  Did you find that all the tests had issues with concurrent high resource usage or was it just one or two classes?  If the latter, you might consider putting this attribute on a single class.

Reply:
It was only a few classes from what I could see, but these tests take less than a minute even when run serially. I'll add a comment.